### ISSUE TITLE
R run configuration shows wrong command line

### ISSUE THREAD

**Issue by rillig** [holgerbrandl/r4intellij#222]
![image](https://user-images.githubusercontent.com/3233724/66971071-09b00c00-f090-11e9-8b39-54c450166fb5.png)

When I run this configuration, the first line in the output says:

> "C:\Users\rillig\Program Files\R-3.6.1\bin\R.exe" --no-save --quiet

That line is missing the `R2pkg_test.R`.

**Comment by holgerbrandl** [comment]
Indeed, it's missing in the output, although it seems to be correctly provided to R.
